src/Illuminate/Foundation/Http/FormRequest.php

```diff
@@ -103,6 +103,16 @@ protected function createDefaultValidator(ValidationFactory $factory)
     }
 
     /**
+     * Get the validated data from the request.
+     *
+     * @return array
+     */
+    public function validated()
+    {
+        return $this->only(array_keys($this->container->call([$this, 'rules'])));
```

Comment:
@JosephSilber why not calling `$this->rules()` directly as the object is already instantiated?